src/iptvsimple/PlaylistLoader.cpp

```diff
@@ -21,7 +21,7 @@
 #include <vector>
 
 #include <kodi/General.h>
-#include <p8-platform/util/StringUtils.h>
+#include "utilities/StringUtils.h"
```

Comment:
As this is no longer a library the include needs to be moved to where the other source file headers are placed.

Reply:
This goes for all the files where this change was made.

Reply:
Fixed, thanks! Principally, is this kind of change OK to be propagated across the rest of PVR add-ons using p8/kodi-platform?